common/test/acceptance/pages/lms/track_selection.py

```diff
@@ -34,24 +34,24 @@ def is_browser_on_page(self):
         """Check if the track selection page has loaded."""
         return self.q(css=".wrapper-register-choose").is_present()
 
-    def enroll(self, mode="honor"):
+    def enroll(self, mode="audit"):
         """Interact with one of the enrollment buttons on the page.
 
             Keyword Arguments:
-                mode (str): Can be "honor" or "verified"
+                mode (str): Can be "audit" or "verified"
```

Comment:
"audit" is not available in the track selection, "audit" is.  Is this incorrect?